core/embed/rust/src/ui/model_tt/layout.rs

```diff
@@ -1652,8 +1655,10 @@ pub static mp_module_trezorui2: Module = obj_module! {
 
     /// def show_spending_details(
     ///     *,
-    ///     account: str,
+    ///     title: str | None,
```

Comment:
It looks like the type here should be rather `title: str = "PIN INFORMATION"`, as the title is in the end always a string, even when you input `None`.

That would require changing the code to 
```rs
let title: StrBuffer = kwargs.get_or(Qstr::MP_QSTR_title, "INFORMATION".into())?
```

The same with `fee_rate_title` below - `: str = "Fee rate:"`

Reply:
Fixed in [c0cfca8](https://github.com/trezor/trezor-firmware/pull/3029/commits/c0cfca8a39ed8bb5da2959741cb3dbdf11deab29).